version.go

```diff
@@ -3,7 +3,7 @@ package posthog
 import "flag"
 
 // Version of the client.
-const Version = "1.2.18"
+const Version = "1.3.0"
```

Comment:
same note here; we auto-bump the version when we deploy.

Reply:
\+